.env

```diff
@@ -1,2 +1,8 @@
 SENTRY_DSN=1234
 CREDIT_COST=5.00
+
+SANDBOX_PAYPAL="AeqFPa09miAc5biGTEqRVUpNdSP6KgaGicgKzphV7-GxzdmigBDxvVF3xVwYGWmStus2REFBvQxiGH8Z"
+
+PRODUCTION_PAYPAL="ARXztybzkFYtYlBeX_MvNPjndIlwXeEJAoZJUtVQTvcTSIaOPZhE_eLeNH3PIR-g2wjDYRFDI4ngyEGH"
```

Comment:
@c-Nosetab The paypal keys shouldn't be checked into github since that would allow anyone to see and use it. We will be including them, along with instructions on how to add them to their local environment, in the handoff document.

Reply:
I think it would be best practice to add the .env file to .gitignore and provide the client with the necessary environment variables so that she/he can add them locally